### PR TITLE
Setup backend

### PR DESCRIPTION
**Pull request by ScottPalme**
Finished setting up the backend. Demo endpoint can be found here: http://127.0.0.1:5000/demo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                    <li>
+                        (terminal): cd .venv
+                    </li>
+                </ul>
+            </li>
+            <li>
+                Activate the enviroment
+                <ul>
+                    <li>
+                        (terminal): source bin/activate
```
Comment (NicoleSauer) [inline README.md:31]: This command only works for mac/linux. 
Windows users need to execute `.\Scripts\Activate.ps1` after changing into the .venv directory.

Comment (ScottPalme) [reply README.md]: Nice hint, i added your suggestion to the readme

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                    </li>
+                </ul>
+            </li>
+            <li>
+                Install the required dependencies:
+                <ul>
+                    <li>
+                        (terminal): pip install -r ../requirements.txt
```
Comment (NicoleSauer) [inline README.md:40]: Again, just a minor addition.
Windows users need to use a backslash: `pip install -r ..\requirements.txt`. Would be nice if you could also add this.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    </li>
+                </ul>
+            </li>
+        </ol>
+    </li>
+    <li>
+        To start the application:
```
Comment (NicoleSauer) [inline README.md:47]: To start the app, one needs to be in the `src` directory. If one ist still in the directory from the previous step, it won´t work. Maybe clarify, that one must switch to `src`.

File: README.md (hunk 2 above)
Comment (ScottPalme) [reply README.md:40]: I added the install command with backslashes

File: README.md (hunk 3 above)
Comment (ScottPalme) [reply README.md:47]: I placed a hint

**Review by NicoleSauer — APPROVED** [review]

**Pull request merged**